optimum/gptq/quantizer.py

```diff
@@ -635,8 +635,12 @@ def load_quantized_model(
         device_map = {"": torch.cuda.current_device()}
         logger.info("The device_map was not initialized." "Setting device_map to `{'':torch.cuda.current_device()}`.")
 
-    with open(os.path.join(save_folder, quant_config_name), "r", encoding="utf-8") as f:
-        quantize_config_dict = json.load(f)
+    # this branch will check if model is from huggingface
+    if hasattr(model, "config") and hasattr(model.config, "quantization_config"):
+        quantize_config_dict = model.config.quantization_config.to_dict()
+    else:
+        with open(os.path.join(save_folder, quant_config_name), "r", encoding="utf-8") as f:
+            quantize_config_dict = json.load(f)
```

Comment:
One last nit, can you raise an error if we were not able to get the quantization config ?

Reply:
Got it, I can make another PR based on this for error handling.

Reply:
We can do it in this PR as it is closely linked ;)

Reply:
sg I will follow up with a commit.